lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -86,9 +90,7 @@ function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
           'crop_rotate' => $values['crop_rotate'],
         ]),
   ];
-
-  $response = drupal_http_request($client['base_url'] . '/reportbacks', $options);
-
+  $response = post($client['base_url'] . '/reportbacks', $options);
```

Comment:
It might help if you reference what we're doing with Northstar in the **dosomethng_northstar.module** file here: https://github.com/DoSomething/phoenix/blob/db40d34f09c1856d3eb5193ff6ed5e8b71eb5e5b/lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module#L26